tests/test_tcp.py

```diff
@@ -9,6 +9,7 @@
 import threading
 import weakref
 
+from OpenSSL import SSL
```

Comment:
Maybe `from OpenSSL import SSL as open_ssl`? :)  To make it obvious where OpenSSL is really used.

Reply:
Sure